File: CodeWars/7kyu/disemvowel-trolls/main.js
function disemvowel(str) {
    no_vowels = str.replace(/a|e|i|o|u/gi, "");
    return no_vowels;
}

const frase_teste = "This website is for losers LOL!";
const frase_resultado = "Ths wbst s fr lsrs LL!";

console.log(disemvowel(frase_teste));
console.log(frase_resultado);

// The replace() method searches a string for a specified value, or a regular expression, and returns a new string where the specified values are replaced.
// string.replace(searchvalue, newvalue)
// g = Perform a global replacement:
// gi = Perform a global, case-insensitive replacement:

File: CodeWars/7kyu/disemvowel-trolls/main.py
def disemvowel(string_):
    vogais = "aeiouAEIOU"
    no_vowels = string_
    for letra in vogais:
        no_vowels = no_vowels.replace(letra, "")
    return no_vowels


frase = "This website is for losers LOL!"
resultado_almejado = "Ths wbst s fr lsrs LL!"
print(disemvowel(frase))
print(resultado_almejado)
# The replace() method replaces a specified phrase with another specified phrase.
